__tests__/cmds/__snapshots__/validate.test.js.snap

```diff
@@ -1,7 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`rdme validate error handling should throw an error if an in valid Swagger definition is supplied 1`] = `
-[Error: Swagger schema validation failed.
+[SyntaxError: Swagger schema validation failed.
```

Comment:
These snapshot updates are because we're throwing the actual error now, rather than extracting the error message and re-throwing it:
Before:
https://github.com/readmeio/rdme/blob/e1745bcb3621bc8c917df3806b62c75b11546ae6/src/cmds/validate.js#L52

After:
https://github.com/readmeio/rdme/blob/337553d71b138567dd45538102667d0448e86ccf/src/lib/prepareOas.js#L18